### DISCUSSION TITLE
Facebook messages not received after integration

### DISCUSSION THREAD

**Discussion by syeddsherazi** [chatwoot/chatwoot#10579] [Self hosted]
After successful facebook channel setup and then inbox creation no messages are received in the inbox.

I did the following steps:

- Followed the Facebook channel setup guide as mentioned here: https://www.chatwoot.com/docs/self-hosted/configuration/features/integrations/facebook-channel-setup
- The channel setup was successful.
- The created a facebook channel as per following steps: https://www.chatwoot.com/hc/user-guide/articles/1677778588-how-to-setup-a-facebook-channel

Can anyone please guide me what might be going wrong?

**Comment by Buda9** [comment]
Same here. I followed tutorial exactly as written and I have same issue

**Reply by amandeepsinghkhalsa** [reply]
@syeddsherazi did you setup webhooks ?
@Buda9 what issue you are facing?
